terminus-ui/autocomplete/src/autocomplete.component.ts

```diff
@@ -85,7 +119,6 @@ const DEFAULT_MINIMUM_CHARACTERS = 2;
  *              selectionsControl="myForm.get('myControl')"
```

Comment:
Looks like a legacy docs mistake, but this should need `[]` wrapping it: `[selectionsControl]...`